checker/tests/ainfer-resourceleak/non-annotated/AddNotOwning.java

```diff
@@ -0,0 +1,89 @@
+import org.checkerframework.checker.calledmethods.qual.*;
```

Comment:
For all these tests in the `non-annotated` folder it's unclear to me what is being tested.  Can you explain how these tests work, or is it documented somewhere?

Reply:
It's documented here: https://github.com/typetools/checker-framework/blob/master/checker/tests/ainfer-README

That document describes the "TestChecker", but for these tests the same process is used with the RLC instead. This README is also symlinked from the ainfer-resourceleak/ directory.